static/app/components/issues/groupListHeader.tsx

```diff
@@ -17,31 +20,70 @@ function GroupListHeader({
   narrowGroups = false,
   withColumns = ['graph', 'event', 'users', 'assignee', 'lastTriggered'],
 }: Props) {
+  const organization = useOrganization();
+
+  const hasNewLayout = organization.features.includes('issue-stream-table-layout');
+
   return (
     <PanelHeader disablePadding>
       <IssueWrapper>{t('Issue')}</IssueWrapper>
-      {withChart && withColumns.includes('graph') && (
-        <ChartWrapper narrowGroups={narrowGroups}>{t('Graph')}</ChartWrapper>
-      )}
-      {withColumns.includes('event') && (
-        <EventUserWrapper>{t('events')}</EventUserWrapper>
-      )}
-      {withColumns.includes('users') && <EventUserWrapper>{t('users')}</EventUserWrapper>}
-      {withColumns.includes('priority') && (
-        <PriorityWrapper narrowGroups={narrowGroups}>{t('Priority')}</PriorityWrapper>
-      )}
-      {withColumns.includes('assignee') && (
-        <AssigneeWrapper narrowGroups={narrowGroups}>{t('Assignee')}</AssigneeWrapper>
-      )}
-      {withColumns.includes('lastTriggered') && (
-        <EventUserWrapper>{t('Last Triggered')}</EventUserWrapper>
+      {hasNewLayout ? (
+        <Fragment>
```

Comment:
Adding the new column headers broke the spacing, so I need to apply updated styles for every column header when the feature flag is on.